sample-apps/go-sample-app/collection/random_metrics.go

```diff
@@ -0,0 +1,171 @@
+package collection
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/instrument/asyncint64"
+	"go.opentelemetry.io/otel/metric/instrument/syncint64"
+)
+
+var (
+	threadsActive int64 = 0
+	threadsBool         = true
+)
+
+// randomMetricCollector contains all the random based metric instruments.
+type randomMetricCollector struct {
+	timeAlive     syncint64.Counter
+	cpuUsage      asyncint64.Gauge
+	heapSize      asyncint64.UpDownCounter
+	threadsActive syncint64.UpDownCounter
+	meter         metric.Meter
+}
+
+// NewRandomMetricCollector returns a new type struct that holds and registers the 4 random based metric instruments used in the Go-Sample-App;
+// HeapSize, ThreadsActive, TimeAlive, CpuUsage
+func NewRandomMetricCollector(mp metric.MeterProvider) randomMetricCollector {
+	rmc := randomMetricCollector{}
+	rmc.meter = mp.Meter(SERVICE_NAME + ".aws-otel-metrics")
```

Comment:
```suggestion
	rmc.meter = mp.Meter("github.com/aws-otel-commnunity/sample-apps/go-sample-app/collection")
```

Same as the `Tracer`, the `Meter` should be named after the import path of the package that creates and uses it.

Reply:
Done